frontend/app/assessments/page.jsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+import React, { useEffect} from 'react'
+import Table from '../components/Table';
+import NavBar from '../components/NavBar';
+import Modal from '../components/Modal';
+
+const Assessments = () => {
+    const surveys = [
```

Comment:
why are the surveys separate from the assessment data?

Reply:
Surveys are the surveys you list when creating the assessment and assessments are the already created assessments that we are listing in the table